src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -125,7 +128,7 @@ public function preSubmit(FormEvent $event)
                 if (!$form->has($name)) {
                     $form->add($name, $this->type, array_replace(array(
                         'property_path' => '['.$name.']',
-                    ), $this->options));
+                    ), $this->getOptionsForEntry($value)));
```

Comment:
But an `add_entry` option would be part of the parent scope (collection type), so could be accessible on in the `ResizeFormListener` which have access to the view data of the field when it adds it, right ?